Lecture6/Task1/Task1L6ApacheHTTP.java

```diff
@@ -0,0 +1,86 @@
+import java.io.IOException;
+
+import CollaboratorClasses.Collaborator;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.http.HttpResponse;
+import org.apache.http.auth.AuthenticationException;
+import org.apache.http.auth.UsernamePasswordCredentials;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.impl.auth.BasicScheme;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.util.EntityUtils;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Task1L6ApacheHTTP {
```

Comment:
Tests shouldn't define how requests are formed. Tests should only execute requests.

Reply:
fixed